hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,7 @@
+RELEASE_TYPE: patch
+
+This patch fixes :issue:`1372`, where :doc: stateful tests <stateful>`
+the sequence of steps to produce an error would only be printed if the
```

Comment:
“where stateful tests the sequence of steps”

perhaps “where in stateful tests, the sequence of steps”?